keystorage/hashicorp/build.gradle

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2019 ConsenSys AG.
+ *
+ * The source code is provided to licensees of PegaSys Plus for their convenience and internal
+ * business use. These files may not be copied, translated, and/or distributed without the express
+ * written permission of an authorized signatory for ConsenSys AG.
+ */
+
+apply plugin: 'java-library'
+apply plugin: 'java-test-fixtures'
+
+jar {
+  archiveBaseName = "hashicorp-vault"
+  manifest {
+    attributes(
+      'Specification-Title': archiveBaseName.get(),
+      'Specification-Version': rootProject.version,
+      'Implementation-Title': archiveBaseName.get(),
+      'Implementation-Version': calculateVersion()
+      )
+  }
+  from {
+    configurations.compile.collect {
+      it.isDirectory() ? it : zipTree(it)
+    }
+  }
+}
+
+dependencies {
+  implementation 'com.google.auto.service:auto-service'
```

Comment:
This shouldn't be needed since we aren't creating a java module

Reply:
done